### ISSUE TITLE
Debug option doesn't work as expected

### ISSUE THREAD

**Issue by sfoxdev** [aquasecurity/trivy#515]
**Description**

Debug option by default should be set to False as described in the `--help`
```
--debug, -d        debug mode (default: false) [$TRIVY_DEBUG]
```

**What did you expect to happen?**
```
docker run --rm -v /var/run/docker.sock:/var/run/docker.sock:ro aquasec/trivy:0.9.0 --no-progress --quiet alpine:3.11.6

alpine:3.11.6 (alpine 3.11.6)
=============================
Total: 0 (UNKNOWN: 0, LOW: 0, MEDIUM: 0, HIGH: 0, CRITICAL: 0)
```
**What happened instead?**
```
docker run --rm -v /var/run/docker.sock:/var/run/docker.sock:ro aquasec/trivy:0.9.0 --no-progress --quiet alpine:3.11.6
2020-06-06T09:52:28.649Z        DEBUG   Severities: UNKNOWN,LOW,MEDIUM,HIGH,CRITICAL

alpine:3.11.6 (alpine 3.11.6)
=============================
Total: 0 (UNKNOWN: 0, LOW: 0, MEDIUM: 0, HIGH: 0, CRITICAL: 0)
```

Another tests:
* Debug False
```
docker run --rm -v /var/run/docker.sock:/var/run/docker.sock:ro aquasec/trivy:0.9.0 -d=false --no-progress --quiet alpine:3.11.6
2020-06-06T09:52:06.357Z        DEBUG   Severities: UNKNOWN,LOW,MEDIUM,HIGH,CRITICAL

alpine:3.11.6 (alpine 3.11.6)
=============================
Total: 0 (UNKNOWN: 0, LOW: 0, MEDIUM: 0, HIGH: 0, CRITICAL: 0)
```
* Debug True
```
docker run --rm -v /var/run/docker.sock:/var/run/docker.sock:ro aquasec/trivy:0.9.0 -d=true --no-progress --quiet alpine:3.11.6

alpine:3.11.6 (alpine 3.11.6)
=============================
Total: 0 (UNKNOWN: 0, LOW: 0, MEDIUM: 0, HIGH: 0, CRITICAL: 0)
```

**Comment by knqyf263** [comment]
@sfoxdev Thank you for reporting the issue. I hope v0.9.1 fixed this problem. If you're still facing the same problem, please feel free to re-open the issue.

**Comment by iamhritik** [comment]
still not fixed facing the same issue in **Version: 0.43.1**

**Comment by knqyf263** [comment]
It works now
